packages/appstore/src/components/banners/wallets-banner/wallets-banner-unsuccessful.tsx

```diff
@@ -2,37 +2,51 @@ import React from 'react';
 import { Icon, Text } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
+import { useDevice } from '@deriv-com/ui';
 
 const WalletsBannerUnsuccessful = observer(() => {
-    const { traders_hub, ui } = useStore();
-    const { is_desktop } = ui;
+    const { traders_hub } = useStore();
+    const { isDesktop, isMobile, isTablet } = useDevice();
     const { toggleWalletsUpgrade } = traders_hub;
+    let titleFontSize, descriptionFontSize, iconHeight, iconWidth;
+
+    if (isMobile) {
+        titleFontSize = 'xs';
+        descriptionFontSize = 'xxxs';
+        iconHeight = '100%';
+        iconWidth = 216;
+    } else if (isTablet) {
+        titleFontSize = 's';
+        descriptionFontSize = 'xxs';
+        iconHeight = 110;
+        iconWidth = 220;
+    } else if (isDesktop) {
+        titleFontSize = 'sm';
+        descriptionFontSize = 'xs';
+        iconHeight = 148;
+        iconWidth = 360;
+    }
```

Comment:
Since these are related, wouldn't it be better to make them an object @nijil-deriv?
```suggestion
   let styles = {};

if (isMobile) {
    styles = {
        titleFontSize: 'xs',
        descriptionFontSize: 'xxxs',
        iconHeight: '100%',
        iconWidth: 216
    };
} else if (isTablet) {
    styles = {
        titleFontSize: 's',
        descriptionFontSize: 'xxs',
        iconHeight: 110,
        iconWidth: 220
    };
} else if (isDesktop) {
    styles = {
        titleFontSize: 'sm',
        descriptionFontSize: 'xs',
        iconHeight: 148,
        iconWidth: 360
    };
}
```

Reply:
Yes, you're right. Thank you 👍